apps/server/src/modules/tldraw/service/tldraw.ws.service.ts

```diff
@@ -206,4 +210,16 @@ export class TldrawWsService {
 	public async flushDocument(docName: string): Promise<void> {
 		await this.tldrawBoardRepo.flushDocument(docName);
 	}
+
+	async authorizeConnection(drawingName: string, token: string) {
+		const headers = {
+			Accept: 'Application/json',
+			Authorization: `Bearer ${token}`,
+		};
+		await firstValueFrom(
```

Comment:
not sure about it, but can you check if this should be use with setInterval ?

Reply:
As I see in the app this is the way to use http calls with this method